src/noise_fns/generators/value.rs

```diff
@@ -121,40 +121,90 @@ impl<T: Float> NoiseFn<Point3<T>, T> for Value {
-        let f1011: T = get(&self.perm_table, [far_corner[0], near_corner[1], far_corner[2], far_corner[3]]);
-        let f0111: T = get(&self.perm_table, [near_corner[0], far_corner[1], far_corner[2], far_corner[3]]);
-        let f1111: T = get(&self.perm_table, [far_corner[0], far_corner[1], far_corner[2], far_corner[3]]);
+        let f0000 = get(
+            &self.perm_table,
+            [
+                near_corner[0],
+                near_corner[1],
+                near_corner[2],
+                near_corner[3],
+            ],
+        );
+        let f1000 = get(
+            &self.perm_table,
+            [
+                far_corner[0],
+                near_corner[1],
+                near_corner[2],
+                near_corner[3],
+            ],
+        );
+        let f0100 = get(
+            &self.perm_table,
+            [
+                near_corner[0],
+                far_corner[1],
+                near_corner[2],
+                near_corner[3],
+            ],
+        );
+        let f1100 = get(&self.perm_table,
+                        [far_corner[0], far_corner[1], near_corner[2], near_corner[3]]);
+        let f0010 = get(
+            &self.perm_table,
+            [
+                near_corner[0],
+                near_corner[1],
+                far_corner[2],
+                near_corner[3],
+            ],
+        );
+        let f1010 = get(&self.perm_table,
+                        [far_corner[0], near_corner[1], far_corner[2], near_corner[3]]);
+        let f0110 = get(&self.perm_table,
+                        [near_corner[0], far_corner[1], far_corner[2], near_corner[3]]);
+        let f1110 = get(&self.perm_table,
+                        [far_corner[0], far_corner[1], far_corner[2], near_corner[3]]);
+        let f0001 = get(
+            &self.perm_table,
+            [
+                near_corner[0],
+                near_corner[1],
+                near_corner[2],
+                far_corner[3],
+            ],
+        );
+        let f1001 = get(&self.perm_table,
+                        [far_corner[0], near_corner[1], near_corner[2], far_corner[3]]);
+        let f0101 = get(&self.perm_table,
+                        [near_corner[0], far_corner[1], near_corner[2], far_corner[3]]);
+        let f1101 = get(&self.perm_table,
+                        [far_corner[0], far_corner[1], near_corner[2], far_corner[3]]);
+        let f0011 = get(&self.perm_table,
+                        [near_corner[0], near_corner[1], far_corner[2], far_corner[3]]);
+        let f1011 = get(&self.perm_table,
+                        [far_corner[0], near_corner[1], far_corner[2], far_corner[3]]);
+        let f0111 = get(&self.perm_table,
+                        [near_corner[0], far_corner[1], far_corner[2], far_corner[3]]);
+        let f1111 = get(&self.perm_table,
+                        [far_corner[0], far_corner[1], far_corner[2], far_corner[3]]);
```

Comment:
Looks like this got mangled by rustfmt. Apparently, the arrays are right around 60 characters long. Running rustfmt with `array_width = 56` breaks all the arrays, which looks better.